### ISSUE TITLE
virtualbox-ose-dkms: DKMS build fails

### ISSUE THREAD

**Issue by wine** [voidlinux/void-packages#11158]
### System

* xuname:  
 ``Void 4.14.15_1 x86_64 AuthenticAMD uptodate r``
* package:  
  ``virtualbox-ose-dkms-5.2.6_1``

### Actual behavior
Building DKMS module 'virtualbox-ose-5.2.6' for kernel-4.14.15_1... FAILED!

``/var/lib/dkms/virtualbox-ose/5.2.6/build/make.log``
```
DKMS make.log for virtualbox-ose-5.2.6 for kernel 4.14.15_1 (x86_64)
Sat Jan 27 13:27:11 CET 2018
make: Entering directory '/usr/src/kernel-headers-4.14.15_1'
  AR      /var/lib/dkms/virtualbox-ose/5.2.6/build/built-in.o
  AR      /var/lib/dkms/virtualbox-ose/5.2.6/build/vboxnetflt/built-in.o
  AR      /var/lib/dkms/virtualbox-ose/5.2.6/build/vboxnetadp/built-in.o
  AR      /var/lib/dkms/virtualbox-ose/5.2.6/build/vboxpci/built-in.o
  AR      /var/lib/dkms/virtualbox-ose/5.2.6/build/vboxdrv/built-in.o
  CC [M]  /var/lib/dkms/virtualbox-ose/5.2.6/build/vboxnetadp/linux/VBoxNetAdp-linux.o
  CC [M]  /var/lib/dkms/virtualbox-ose/5.2.6/build/vboxnetflt/linux/VBoxNetFlt-linux.o
cc1: error: incompatible gcc/plugin versions
cc1: error: fail to initialize plugin ./scripts/gcc-plugins/structleak_plugin.so
cc1: error: incompatible gcc/plugin versions
cc1: error: fail to initialize plugin ./scripts/gcc-plugins/latent_entropy_plugin.so
cc1: error: incompatible gcc/plugin versions
cc1: error: fail to initialize plugin ./scripts/gcc-plugins/structleak_plugin.so
cc1: error: incompatible gcc/plugin versions
cc1: error: fail to initialize plugin ./scripts/gcc-plugins/latent_entropy_plugin.so
  CC [M]  /var/lib/dkms/virtualbox-ose/5.2.6/build/vboxpci/linux/VBoxPci-linux.o
  CC [M]  /var/lib/dkms/virtualbox-ose/5.2.6/build/vboxdrv/linux/SUPDrv-linux.o
cc1: error: incompatible gcc/plugin versions
cc1: error: fail to initialize plugin ./scripts/gcc-plugins/structleak_plugin.so
cc1: error: incompatible gcc/plugin versions
cc1: error: fail to initialize plugin ./scripts/gcc-plugins/latent_entropy_plugin.so
cc1: error: incompatible gcc/plugin versions
cc1: error: fail to initialize plugin ./scripts/gcc-plugins/structleak_plugin.so
cc1: error: incompatible gcc/plugin versions
cc1: error: fail to initialize plugin ./scripts/gcc-plugins/latent_entropy_plugin.so
make[2]: *** [scripts/Makefile.build:321: /var/lib/dkms/virtualbox-ose/5.2.6/build/vboxnetadp/linux/VBoxNetAdp-linux.o] Error 1
make[1]: *** [scripts/Makefile.build:579: /var/lib/dkms/virtualbox-ose/5.2.6/build/vboxnetadp] Error 2
make[1]: *** Waiting for unfinished jobs....
make[2]: *** [scripts/Makefile.build:321: /var/lib/dkms/virtualbox-ose/5.2.6/build/vboxnetflt/linux/VBoxNetFlt-linux.o] Error 1
make[1]: *** [scripts/Makefile.build:579: /var/lib/dkms/virtualbox-ose/5.2.6/build/vboxnetflt] Error 2
make[2]: *** [scripts/Makefile.build:321: /var/lib/dkms/virtualbox-ose/5.2.6/build/vboxdrv/linux/SUPDrv-linux.o] Error 1
make[1]: *** [scripts/Makefile.build:579: /var/lib/dkms/virtualbox-ose/5.2.6/build/vboxdrv] Error 2
make[2]: *** [scripts/Makefile.build:321: /var/lib/dkms/virtualbox-ose/5.2.6/build/vboxpci/linux/VBoxPci-linux.o] Error 1
make[1]: *** [scripts/Makefile.build:579: /var/lib/dkms/virtualbox-ose/5.2.6/build/vboxpci] Error 2
make: *** [Makefile:1507: _module_/var/lib/dkms/virtualbox-ose/5.2.6/build] Error 2
make: Leaving directory '/usr/src/kernel-headers-4.14.15_1'
```

**Comment by Piraty** [comment]
This also affects wireguard ( and possibly some more kernel modules?).
It is because the kernel is built with a different gcc version than we all have on our systems (as was said on IRC)

**Comment by chneukirchen** [comment]
Perhaps these GCC plugins are more trouble than they are worth.

**Comment by Dostoevskij** [comment]
Had thevsame problem with nvidia-dkms, it's fixed with 4.14.15_2

**Comment by wine** [comment]
Confirmed, fixed with fe296b233512d2286dd15b7bed127f93aa34614a
